css/amp.css

```diff
@@ -284,13 +299,19 @@ i-amphtml-sizer {
   /* For author styling. */
 }
 
-.i-amphtml-notbuilt {
+.i-amphtml-notbuilt,
+[layout]:not(.i-amphtml-element),
+.-amp-element:not(.i-amphtml-element)
+{
   position: relative;
   overflow: hidden !important;
   color: transparent !important;
 }
 
-.i-amphtml-notbuilt:not(.i-amphtml-layout-container) > * {
+.i-amphtml-notbuilt:not(.i-amphtml-layout-container) > * ,
+[layout]:not(.i-amphtml-element):not(.i-amphtml-layout-container) > *,
```

Comment:
Perhaps `[layout]:not([layout="container"]):{the rest of what you have}`?

Reply:
👍